lib/core/application/post/create_post_bloc/create_post_bloc.dart

```diff
@@ -79,8 +80,12 @@ class CreatePostBloc extends Cubit<CreatePostState> {
     );
     response.fold(
       (l) => emit(state.copyWith(status: CreatePostStatus.error)),
-      (isUpdateSuccess) =>
-          emit(state.copyWith(status: CreatePostStatus.postCreated)),
+      (postDto) => emit(
+        state.copyWith(
+          status: CreatePostStatus.postCreated,
+          newPost: Post.fromDto(postDto),
```

Comment:
the convert from dto to entities should be handled in repository not in bloc

Reply:
Yep, convert from dt to entited shouldn't handle here 👍